man/bundle-pristine.ronn

```diff
@@ -8,6 +8,29 @@ bundle-pristine(1) -- Restores installed gems to their pristine condition
 ## DESCRIPTION
 
 `pristine` restores the installed gems in the bundle to their pristine condition using the local gem
-cache from RubyGems. For git gems - a forced checkout will be performed.
+cache from RubyGems.
+For git gems, a forced checkout will be performed.
+`bundle pristine` ignores unpacked files on disk. This command
```

Comment:
This paragraph is really start and stop, i suggest maybe just tinkering with this to help with readability